core/src/main/java/com/yahoo/tagchowder/Parser.java

```diff
@@ -166,6 +168,11 @@
      **/
     public static final String VALIDATION_FEATURE = "http://xml.org/sax/features/validation";
 
+    /**
+     * Controls whether the parser is reporting all validity errors for AMP contents.
+     */
+    public static final String AMP_VALIDATION_FEATURE = "http://xml.org/sax/features/amp-validation";
```

Comment:
if this is just a dummy string, we might as well put the amp java validator git link

Reply:
Done